better_than_this/http_funcs.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
We probably want to rename this file something like `yelp_api_client.py` in the future. Good for now though!